fe/src/main/java/org/apache/doris/analysis/CreateRoutineLoadStmt.java

```diff
@@ -153,122 +178,129 @@ public int getDesiredConcurrentNum() {
         return desiredConcurrentNum;
     }
 
-    public int getMaxErrorNum() {
+    public long getMaxErrorNum() {
         return maxErrorNum;
     }
 
-    public String getKafkaEndpoint() {
-        return kafkaEndpoint;
+    public long getMaxBatchIntervalS() {
+        return maxBatchIntervalS;
+    }
+
+    public long getMaxBatchRows() {
+        return maxBatchRows;
+    }
+
+    public long getMaxBatchSize() {
+        return maxBatchSizeBytes;
+    }
+
+    public String getKafkaBrokerList() {
+        return kafkaBrokerList;
     }
 
     public String getKafkaTopic() {
         return kafkaTopic;
     }
 
-    public List<Integer> getKafkaPartitions() {
-        return kafkaPartitions;
+    public List<Pair<Integer, Long>> getKafkaPartitionOffsets() {
+        return kafkaPartitionOffsets;
     }
 
     @Override
-    public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
+    public void analyze(Analyzer analyzer) throws UserException {
         super.analyze(analyzer);
+        // check dbName and tableName
+        checkDBTable(analyzer);
         // check name
```

Comment:
The checkDBTable throws AnalysisException, but analyze does't catch it or throws it too.

Reply:
It will throw analysisException to outside.